SPECS/lynx/lynx.spec

```diff
@@ -16,17 +16,16 @@
 #
 
 
-%define tarbase 2.9.0dev.9
 Summary:        A Text-Based WWW Browser
 Name:           lynx
-Version:        2.9.0~dev.9
-Release:        5%{?dist}
+Version:        2.9.0
```

Comment:
As per the test pipeline, there seems to be a dependency on `libcrypto`.
Can we once check if it is just the pipeline issue or an actual issue with this package ?